Makefile

```diff
@@ -101,9 +101,12 @@ docker-test: docker-build
 .PHONY: docker-build docker-build-release docker-run docker-test
 
 # --- Deployment -------------------------------------------------------------------
+LOCAL_OVERLAY = deployment/$*/overlay.jsonnet
+REMOTE_OVERLAY = https://github.com/foxygoat/foxtrot/raw/$(REF)/$(LOCAL_OVERLAY)
```

Comment:
For future thought: We should extract out this repo name (and probably add it to the Version struct too). Then this remote overlay becomes able to be automatically generated: `$(GITHUB_REPO)/raw/$(REF)/$(LOCAL_OVERLAY)` - that would always be the same for every repo.

Reply:
yeah. I added the repo url to jcdc.
but then I usually look at the version info in the browser and I don't pretty print it and it becomes less readable.
so I decided we should probably just keep the app name only cause _we_ know where the repos are.

the right thing would be a blueprint type file somewhere in the repo - embeddable?? that has app name and repo name as discussed elsewhere.